Unreal/CarlaUE4/Plugins/Carla/Source/Carla/Sensor/CollisionSensor.cpp

```diff
@@ -32,27 +35,37 @@ void ACollisionSensor::SetOwner(AActor *NewOwner)
   Super::SetOwner(NewOwner);
 
   /// @todo Deregister previous owner if there was one.
-
   if (NewOwner != nullptr)
```

Comment:
**Code Simplification Suggestion**
This null check if could be removed as we are already checking now if the pointer is valid below.

Reply:
You are right, I'll do it!

Reply:
Applied